templates/bundles/DemosPlanCoreBundle/DemosPlanStatement/new_public_participation_statement_confirm_entry.html.twig

```diff
@@ -1,64 +1,66 @@
-            </dt><!--
-         --><dd class="{{ 'layout__item u-1-of-3 u-ml-0-lap-up u-1-of-1-palm u-ml-0_5-palm'|prefixClass }}">
-                {% if statement.polygon|default != '' %}
-                    <button
-                        class="{{ 'btn--blank o-link--default'|prefixClass }}"
-                        type="button"
-                        @click.prevent.stop="(function(){$refs.mapModal.toggleModal({{ statement.polygon|convertLegacyPolygon|escape('html_attr') }})})()">
-                        {{ 'see'|trans }}
-                    </button>
-                {% else %}
-                    {{ "notspecified"|trans }}
-                {% endif %}
-            </dd><!--
-        {% endif %}
-
-        {% if hasPermission('feature_documents_new_statement') %}
-        --><dt class="{{ 'layout__item u-1-of-6 weight--bold u-1-of-1-palm'|prefixClass }}">
-                {{ "paragraph"|trans }}:
-            </dt><!--
-        --><dd class="{{ 'layout__item u-1-of-3 u-ml-0-lap-up u-1-of-1-palm u-ml-0_5-palm'|prefixClass }}">
-                {% if statement.paragraph.title is defined %}{{ statement.paragraph.title|striptags|default() }}{% endif %}
-            </dd><!--
-        {% endif %}
-
-     --><dt class="{{ 'layout__item u-1-of-6 weight--bold u-1-of-1-palm'|prefixClass }}">
-            {{ "file"|trans }}:
-        </dt><!--
-     --><dd class="{{ 'layout__item u-1-of-3 u-ml-0-lap-up o-hellip u-1-of-1-palm u-ml-0_5-palm'|prefixClass }}">
-            {% if statement.files|default([])|length > 0 %}
-                {% for file in statement.files %}
-                <a
-                    target="_blank"
-                    rel="noopener"
-                    href="{{ path("core_file", { 'hash': file|getFile('hash') }) }}" title="{{ file|getFile('name') }}">
-                    {{ file|getFile('name') }}
-                </a><br>
-                {% endfor %}
+            {% if hasPermission('field_statement_location') %}
+                <dt class="{{ 'col-start-1 mb-1 weight--bold '|prefixClass }}">
+                    {{ "location"|trans }}:
+                </dt>
+                 <dd class="{{ 'col-start-2 ml-7 md:ml-5 lg:ml-3'|prefixClass }}">
+                    {% if statement.polygon|default != '' %}
+                        <button
+                            class="{{ 'btn--blank o-link--default'|prefixClass }}"
+                            type="button"
+                            @click.prevent.stop="(function(){$refs.mapModal.toggleModal({{ statement.polygon|convertLegacyPolygon|escape('html_attr') }})})()">
+                            {{ 'see'|trans }}
+                        </button>
+                    {% else %}
+                        {{ "notspecified"|trans }}
+                    {% endif %}
+                </dd>
             {% endif %}
-        </dd>
-    </dl>
+        </dl>
 
+        <dl class="{{ 'grid grid-cols-8 w-full md:w-1/2 mb-0'|prefixClass }}">
```

Comment:
Can't we apply the same layout here as in the "Meine Daten" section ("Datei" below "Dokument"? I think that would be more consistent and it would prevent the 'Datei' field from appearing detached. 
Also, while we're at it, we could make the spacing below the headings more consistent, so that "Stellungnahme mitzeichnen" has the same space below as the "Meine Daten" heading.

Reply:
thx for the idea, done here https://github.com/demos-europe/demosplan-core/pull/2747/commits/3e984a79f52dc83711d398bc3e20d9f6abdced01